sphinxcontrib/openapi.py

```diff
@@ -122,10 +122,30 @@ def _httpresource(endpoint, method, properties):
     yield ''
 
 
-def openapi2httpdomain(spec):
+def _clean_paths_option(value):
```

Comment:
It seems like this function is general one, and is aimed to convert a string value into a list of values, regardless of the option. Hence, I think it would make sense if you rename it into one of the following:

* `_multioption`
* `_listoption`
* `_multiple_option`
* `_multivalue`
* `_listvalue`

Honestly I'm not sure about the naming. Taking into account that docutils defines `positive_int_list`, maybe it might make sense to name it accordingly: `string_list`? What's your opinion on this?